### ISSUE TITLE
Chaining graylog servers with GELF output leads to exception in the receiver

### ISSUE THREAD

**Issue by kroepke** [Graylog2/graylog2-server#1198]
If you chain two Graylog servers, beginning in 1.1.0-rc.1, the receiver allows overriding the reserved settable fields in a GELF message.

Unfortunately the gl2_source_input cannot be set when receiving a message, which leads to an exception when decoding the message:

```
2015-06-01 09:44:39,772 WARN : org.graylog2.shared.buffers.ProcessBuffer - Unable to process event MessageEvent{raw=null, message=null}, sequence 534 
java.lang.IllegalStateException: Multiple server nodes 
at org.graylog2.shared.buffers.processors.DecodingProcessor.processMessage(DecodingProcessor.java:151) 
at org.graylog2.shared.buffers.processors.DecodingProcessor.onEvent(DecodingProcessor.java:79) 
at org.graylog2.shared.buffers.processors.ProcessBufferProcessor.onEvent(ProcessBufferProcessor.java:56) 
at org.graylog2.shared.buffers.processors.ProcessBufferProcessor.onEvent(ProcessBufferProcessor.java:33) 
at com.lmax.disruptor.WorkProcessor.run(WorkProcessor.java:138) 
at com.codahale.metrics.InstrumentedExecutorService$InstrumentedRunnable.run(InstrumentedExecutorService.java:176) 
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) 
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) 
at java.lang.Thread.run(Thread.java:745)
```

**Comment by bernd** [comment]
I just reverted be47150c89f71655f99f0d751e556a8c00b22125. Now we need to find a solution for Graylog2/graylog2-web-interface#1334.

**Comment by bernd** [comment]
Fixed in 611f9a8.
